{"timestamp":"2025-10-05T12:40:41.616190Z","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager","filename":"manager.py","lineno":179}
{"timestamp":"2025-10-05T12:40:41.616608Z","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/my-first-dag.py","logger":"airflow.models.dagbag.DagBag","filename":"dagbag.py","lineno":593}
{"timestamp":"2025-10-05T12:40:41.644171Z","level":"warning","event":"The `airflow.operators.python.PythonOperator` attribute is deprecated. Please use `'airflow.providers.standard.operators.python.PythonOperator'`.","category":"DeprecatedImportWarning","filename":"/opt/airflow/dags/my-first-dag.py","lineno":2,"logger":"py.warnings"}
{"timestamp":"2025-10-05T12:40:41.668920Z","level":"info","event":"Task instance is in running state","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:40:41.669027Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:40:41.669177Z","level":"info","event":"Current task name:extract","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:40:41.669238Z","level":"info","event":"Dag name:my_first_python_etl_dag","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:40:41.671032Z","level":"info","event":"Extraction Phase","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:40:41.671482Z","level":"error","event":"Task failed with exception","logger":"task","filename":"task_runner.py","lineno":994,"error_detail":[{"exc_type":"PermissionError","exc_value":"[Errno 13] Permission denied: '../extracted-data.txt'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1307,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":416,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":82,"name":"run"},{"filename":"/opt/airflow/dags/my-first-dag.py","lineno":17,"name":"extract"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-05T12:40:41.690287Z","level":"info","event":"Task instance in failure state","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:40:41.690456Z","level":"info","event":"Task start","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:40:41.690514Z","level":"info","event":"Task:<Task(PythonOperator): extract>","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:40:41.690548Z","level":"info","event":"Failure caused by [Errno 13] Permission denied: '../extracted-data.txt'","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:40:41.736188Z","level":"warning","event":"Using Connection.get_connection_from_secrets from `airflow.models` is deprecated.Please use `get` on Connection from sdk(`airflow.sdk.Connection`) instead","category":"DeprecationWarning","filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py","lineno":486,"logger":"py.warnings"}
{"timestamp":"2025-10-05T12:40:41.743975Z","level":"info","event":"Email alerting: attempt 1","logger":"airflow.utils.email","filename":"email.py","lineno":261}
{"timestamp":"2025-10-05T12:40:41.748911Z","level":"info","event":"Email alerting: attempt 1","logger":"airflow.utils.email","filename":"email.py","lineno":261}
{"timestamp":"2025-10-05T12:40:41.749173Z","level":"error","event":"Top level error","logger":"task","filename":"task_runner.py","lineno":1457,"error_detail":[{"exc_type":"ConnectionRefusedError","exc_value":"[Errno 111] Connection refused","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1452,"name":"main"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1419,"name":"finalize"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1259,"name":"_send_task_error_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":79,"name":"send_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":154,"name":"send_email_smtp"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":263,"name":"send_mime_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":306,"name":"_get_smtp_connection"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":255,"name":"__init__"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":341,"name":"connect"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":312,"name":"_get_socket"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":865,"name":"create_connection"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":850,"name":"create_connection"}],"is_group":false,"exceptions":[]},{"exc_type":"ConnectionRefusedError","exc_value":"[Errno 111] Connection refused","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1257,"name":"_send_task_error_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":79,"name":"send_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":154,"name":"send_email_smtp"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":263,"name":"send_mime_email"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py","lineno":306,"name":"_get_smtp_connection"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":255,"name":"__init__"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":341,"name":"connect"},{"filename":"/usr/python/lib/python3.12/smtplib.py","lineno":312,"name":"_get_socket"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":865,"name":"create_connection"},{"filename":"/usr/python/lib/python3.12/socket.py","lineno":850,"name":"create_connection"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-05T12:40:41.759059Z","level":"warning","event":"Process exited abnormally","exit_code":1,"logger":"task"}
